### PR TITLE
fix: fix error message on importing/reading a package instead of a module

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

File: core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go [hunk 1]
```diff
@@ -293,7 +293,8 @@ func getReferenceName(repo *git.Repository, parsedURL *ParsedGitURL) (plumbing.R
 	return "", false, nil
 }
 
-/**
+/*
+*
```
Comment (gbouv) [inline core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go]: Is that a legit go comment formatting here?? 🤔 
I personally prefer what we had before...

File: core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func (provider *GitPackageContentProvider) GetOnDiskAbsoluteFilePath(fileInsideP
 		return "", interpretationError
 	}
 	if parsedURL.relativeFilePath == "" {
-		return "", startosis_errors.NewInterpretationError("The relative path to file is empty for '%v'", fileInsidePackageUrl)
+		return "", startosis_errors.NewInterpretationError("Path to import '%v' needs to point to a specific file but didn't. Users can only read or import specific files and not entire packages.", fileInsidePackageUrl)
```
Comment (gbouv) [inline core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go]: nit: the phrasing looks weird to me. Though I am not a native english speaker. Something like "_The path_ '%s' needs to point to a specific file but _it_ didn't" resonates better to me

**Review by gbouv — APPROVED** [review]

File: core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go (hunk 1 above)
Comment (h4ck3rk3y) [reply core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go]: That was auto, but reverted.

File: core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go (hunk 2 above)
Comment (h4ck3rk3y) [reply core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/git_package_content_provider.go]: Agreed, fixed.

**Pull request merged**